### ISSUE TITLE
Find>Search all Sketch Tabs(checked)>Replace All - only applies to the current tab and tabs to the right of the current tab

### ISSUE THREAD

**Issue by per1234** [arduino/Arduino#2402]
Arduino IDE 1.5.8
Windows 7 64 bit

<b>What steps will reproduce the problem?

open sketch with multiple tabs
enter text to be replaced in one of the tabs
select a tab to the right of that tab
Find
enter the text to be replaced in the Find: field
enter different text in the Replace With: field
check Search all Sketch Tabs
Replace All

<b>What is the expected output?

all instances of the text in the Find: field in all tabs of the sketch should be replaced

<b>What do you see instead?

only instances of the text in the Find: field in the current tab and tabs to the right of the current tab are replaced.

**Comment by per1234** [comment]
This bug still exists in Arduino IDE 1.6.0

**Comment by swordmaster2k** [comment]
I can confirm this is an issue with 1.6.0. Note that it only effects Find All.

**Comment by swordmaster2k** [comment]
I have identified the cause of this issue after a little debugging. The Replace All is starting at the currently selected tabs index and iterating through all of the tabs to the right.

It does not take into account that there may be any preceding tabs. I'm looking into fixing this bug so I've forked the project and branched specifically for this issue.

When Replace All is used across multiple tabs it should probably start at the first tab's index in the sketch rather than the currently selected.
